src/current/v23.2/live-migration-service.md

```diff
@@ -670,6 +750,116 @@ These steps assume you have already followed the overall steps to [prepare for m
 1. Any writes that were made during the cutover will have been missed on CockroachDB. Use [MOLT Verify]({% link {{ page.version.version }}/molt-verify.md %}) to identify the inconsistencies. These will need to be manually reconciled.
 {% endcomment %}
 
+## Monitor cutover
```

Comment:
Have you considered making this an H3 under the previous section? Maybe you decided against it to avoid having H4s, but it seems a little odd that you might monitor a cutover you didn't initiate.

Reply:
I did go back and forth on that -- not sure why I settled on an H2. I agree with you.